modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -271,13 +269,13 @@ function AddFDSFilterModalContent({
 		fieldNames?.includes(item.name) ? item.name : undefined
 	);
 
-	const CellRendererDropdown = ({
-		cellRenderers,
+	const FieldNameDropdown = ({
```

Comment:
This is close candidate be turned into a app-level component that can be used both for cell renderers and for filters. I'm more than fine to have them separated for now, just note it down for future evaluation

Reply:
It should be changed by Clay Picker soon, but the name was confusing